app/code/Magento/Bundle/Ui/DataProvider/Product/Form/Modifier/BundlePrice.php

```diff
@@ -94,7 +97,10 @@ public function modifyMeta(array $meta)
     }
 
     /**
-     * {@inheritdoc}
```

Comment:
Please, revert inheritdoc annotation

Reply:
Reverting @inheritdoc will fail the build and also https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html

@inheritdoc tag
The @inheritdoc tag SHOULD NOT be used. If a child class method requires a long description to explain its purpose, it may use @inheritdoc to indicate the new description is intended as an addition to the parent method description. In general such method overrides are a code smell and should be used as an incentive to make the code more self-documenting if possible.

As per our function description need not to be very big so i removed. Suggest me if thats a wrong way.

Reply:
It's true for the new code, in the legacy code we still use it, just replace {@inheritdoc} with @inheritdoc